packages/@aws-cdk/aws-appsync/lib/graphqlapi.ts

```diff
@@ -1027,6 +1066,128 @@ export abstract class MappingTemplate {
     return this.fromString('$util.toJson($ctx.result)');
   }
 
+  /**
```

Comment:
Lets limit the request mapping templates to `httpGet`, `httpPost`, `httpPatch`, and `httpDelete`. We can take additional arguments to allow the user to specify whether they want to pass headers/body.

Reply:
I think giving an example in README is enough for now. I removed them.